gc/base/MemorySubSpaceSemiSpace.cpp

```diff
@@ -509,20 +510,18 @@ MM_MemorySubSpaceSemiSpace::flip(MM_EnvironmentBase *env, Flip_step step)
 			_memorySubSpaceAllocate = _memorySubSpaceEvacuate;
 			_memorySubSpaceEvacuate = _memorySubSpaceSurvivor;
 			getMemorySpace()->setDefaultMemorySubSpace(getDefaultMemorySubSpace());
-			if (debug) {
-				omrtty_printf("tilt backout _allocateSpaceBase/Top %llx/%llx _survivorSpaceBase/Top %llx/%llx tilt sizes %llx %llx\n",
-						_allocateSpaceBase, _allocateSpaceTop, _survivorSpaceBase, _survivorSpaceTop,
-						(uintptr_t)_allocateSpaceTop - (uintptr_t)_allocateSpaceBase +  (uintptr_t)_survivorSpaceTop - (uintptr_t)_survivorSpaceBase, (uintptr_t)0);
-			}
+			Trc_MM_MSSSS_flip_backout(env->getLanguageVMThread(),
+					"",
+					_allocateSpaceBase, _allocateSpaceTop, _survivorSpaceBase, _survivorSpaceTop,
+					(uintptr_t)_allocateSpaceTop - (uintptr_t)_allocateSpaceBase +  (uintptr_t)_survivorSpaceTop - (uintptr_t)_survivorSpaceBase, (uintptr_t)0);
```

Comment:
Do we need the `(uintptr_t)` cast with every parameter? Can this line of code be simplified?

Reply:
they are 'void *' to start with, so not that much that we can simplify away

in more general sense, we could convert pointers (something like char * or uint *), but it would go beyond the scope of the change (affecting other usages of these pointers)